tests/test_xero/test_utils.py

```
@@ -473,3 +473,77 @@ def test_post_payment(mocker, db):
     )
 
     assert created_payment == []
+
+
+def test_is_duplicate_deletion_skipped(db):
+    workspace_id = 1
+    xero_credentials = XeroCredentials.get_active_xero_credentials(workspace_id=workspace_id)
+    xero_connection = XeroConnector(credentials_object=xero_credentials, workspace_id=workspace_id)
+    
+    # Test for attribute types that should not skip deletion
+    assert xero_connection.is_duplicate_deletion_skipped('ACCOUNT') == False
```

Ruff (0.11.9)

Avoid equality comparisons to `False`; use `if not ...:` for false checks

Replace comparison

(E712)